soc/xtensa/espressif_esp32/common/Kconfig.soc

```diff
@@ -37,6 +37,14 @@ config ESP_HEAP_SEARCH_ALL_REGIONS
 	  regions. If the region of desired capability is exhausted,
 	  memory will be allocated from other available region.
 
+config SPIRAM_HEAP_SIZE
+	int "Size of SPIRAM heap"
+	default 262134 if CONFIG_SYS_HEAP_SMALL_ONLY
```

Comment:
Only `SYS_HEAP_SMALL_ONLY` ;)

Reply:
Thanks for catching that.  I wrote that originally and then fixed it when I was testing and apparently forgot to squash my fixes before the upload :(  Will retest and push again.

Reply:
Fixed this and tested on the same branch that I pushed this time.